pkg/services/accesscontrol/scope_test.go

```diff
@@ -118,6 +122,20 @@ func TestScopeResolver_ResolveAttribute(t *testing.T) {
 			),
 			wantCalls: 2,
 		},
+		{
+			name:          "should resolve name with colon",
+			orgID:         1,
+			evaluator:     EvalPermission("datasources:read", Scope("datasources", "name", "test:ds4")),
+			wantEvaluator: EvalPermission("datasources:read", Scope("datasources", "id", "4")),
```

Comment:
nit: I think when @kalleep's changes from https://github.com/grafana/grafana/pull/46741 are merged, we'll be resolving from name to UID (instead of name -> ID). So this will either need to be changed in Kalle's PR or this PR, depending on how we merge.